src/scripts/buttonClickHandlers.js

```diff
@@ -0,0 +1,124 @@
+import { getSavedArray, changeSavedArray } from './utilityFunctions';
+import { renderEditForm } from './renderEditForm';
+
+const getItemIndex = (clickedItem, array) => {
+    return Array.from(array).indexOf(clickedItem);
```

Comment:
Nazwa array jest dla mnie nieadekwatna, jeżeli jest to NodeList, to nie lepiej
```suggestion
    return[...array].indexOf(clickedItem);
```